sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -141,6 +142,15 @@ def test_basic_workflow(self):
       shutil.rmtree(tmpdir)
       # print(tmpdir)
 
+  def test_convert_k8s_obj_to_dic(self):
+    converted = compiler.Compiler()._convert_k8s_obj_to_dic({
```

Comment:
The argument given to `_convert_k8s_obj_to_dic` is not a k8s object. What exactly is this call trying to show?